guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -7,73 +7,89 @@ You can register a host by using registration templates and set up various integ
+This is required to install the `insights-client` package on the host.
 endif::[]
 include::snip_prerequisite-configured-smart-proxy-registration-provisioning.adoc[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
 . Optional: Select a different *Organization*.
 . Optional: Select a different *Location*.
-. Optional: From the *Host Group* list, select the host group to associate the hosts with.
+. Optional: From the *Host Group* list, select the host group to associate the host with.
 Fields that inherit value from *Host group*: *Operating system*, *Activation Keys* and *Lifecycle environment*.
-. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
+. Optional: From the *Operating system* list, select the operating system of the host that you want to register.
 ifndef::satellite,orcharhino[]
 Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
 endif::[]
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
+. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register the host through.
 ifdef::katello,satellite,orcharhino[]
-. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
+. In the *Activation Keys* field, enter one or more activation keys to assign to the host.
 endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
-During this first call, hosts download the CA file from {Project}.
-Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+. Optional: If you want to use `wget` to register the host to {Project}, select the *Use wget* option.
+By default, {Project} generates a `curl` command.
+. Optional: If you want to make the first call insecure, select the *Insecure* option.
+During this first call, the host downloads the CA file from {Project}.
+The host will use this CA file to connect to {Project} with all future calls making them secure.
 +
 {Team} recommends that you avoid insecure calls.
 +
-If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
+If an attacker, located in the network between {Project} and the host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
 Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
 +
 Instead, you can manually copy and install the CA file on each host before registering the host.
```

Comment:
```suggestion
Therefore, it is highly recommended to manually copy and install the CA file on each host before registering them instead of using SSH keys.
```
Not a part of this PR but this should be precise.

Reply:
This is an incorrect suggestion. The SSH keys are related to remote execution. The user can deploy them regardless of using or not using the insecure call. It just says that these SSH keys can be stolen if the call is insecure.

Reply:
How about we make it more specific `Therefore in this case,....` 

I mean I am not overlooking SSH. But, particular to this case only.

Reply:
But that doesn't help. It's already specific enough.

Reply:
Alright, we can disregard this 👍